src/cpp/include/loss.h

```diff
@@ -0,0 +1,82 @@
+#ifndef MARIUS_LOSS_H
+#define MARIUS_LOSS_H
+
+#include <torch/torch.h>
```

Comment:
Follow the google style guide for header imports: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes